BlueprintUICommonControls/Sources/AccessibilityContainer.swift

```diff
@@ -0,0 +1,53 @@
+import BlueprintUI
+import UIKit
+
+/// Acts as an accessibility container for any subviews
+/// where `isAccessibilityElement == true`.
+public struct AccessibilityContainer: Element {
```

Comment:
Would be good to flesh this comment out a bit more

Reply:
Agreed, I updated the behavior and rounded out the doc comment more; open to any improvements to the wording, describing "computer science" is not my forte